src/renderer/components/BouncingSlippiLogo.tsx

```diff
@@ -23,13 +28,18 @@ const Outer = styled.div<{
 
 const Logo = styled.div<{
   size: string;
+  roll: boolean;
 }>`
   background-image: url("${slippiLogo}");
   background-size: contain;
   background-repeat: no-repeat;
-  animation: ${bounceAnimation} 1.2s infinite;
+  animation: ${bounceAnimation} 1.2s infinite forwards,
+    ${barrelRollAnimation} 1s cubic-bezier(0.68, -0.55, 0.265, 1.55) alternate forwards;
+  animation-play-state: running, ${(props) => (props.roll ? "running" : "paused")};
   position: absolute;
-  height: ${(props) => props.size};
+  // logo is proportionally smaller to size by a factor of 0.75
+  // so we need to scale it down to avoid pivotal animations
```

Comment:
```suggestion
```
I think I would prefer to keep comments outside of the `css` definition itself. If you want to comment maybe add it as javascript comments above the definition.